README.md

```diff
@@ -74,42 +74,28 @@ Usage: cyclonedx-npm [options] [--] [<package-manifest>]
 Create CycloneDX Software Bill of Materials (SBOM) from Node.js NPM projects.
 
 Arguments:
-  <package-manifest>        Path to project's manifest file.
-                            (default: "package.json" file in current working directory)
+  <package-manifest>        Path to project's manifest file. (default: "package.json" file in current working directory)
```

Comment:
please keep the formatting that was used in the README.
the line breaks that were added for readability need to stay.